### ISSUE TITLE
Issue with Player Join & Player Leave Regex - Arma Reforger

### ISSUE THREAD

**Issue by sxon-xo** [CubeCoders/AMPTemplates#683]
There is an issue with the Player Join and Player Leave Regex for Arma Reforger. It is not counting joins and leaves—a possible regex change due to the update.

I have tested the below settings for join and leave, and they seem to be working if we update this. If you have any ideas, please let me know.

`^[\d:\s]*Player` #\d+ (?<username>.+) \(\d+\.\d+\.\d+\.\d+:\d+\) `connected$`
`^[\d:\s]*Player` #\d+ (?<username>.+) `disconnected$`

Console Logs:

`Player` #0 SXon (IP:PORT GOES HERE) `connected`
`Player` #0 SXon `disconnected`

Thank you :)

**Comment by Greelan** [comment]
Thanks. Is that logging the same on both windows and Linux servers?

It would be helpful to have the full logs around when players join and leave rather than just single lines. Can you share?

**Comment by sxon-xo** [comment]
I can't speak for Windows as I don't use it in my environment - this is for Linux running the instance in Docker.

Here are the full logs.


`DEFAULT      : BattlEye Server: RCon admin #0 (IP:PORT) logged in
SCRIPT       : ScenarioFramework: -> LayerTask: SlotTask SlotDestroy_19 - generating task Destroy vehicle. Description: ScenarioFramework: -> LayerTask: SlotTask SlotDestroy_19 - generating task Destroy vehicle. Description: 20 Command Truck
RPL          : ServerImpl event: authenticating (identity=0x00000000, address=IP:PORT)
RPL          : ServerImpl event: connected (identity=0x00000000)
WORLD        : UpdateEntities
Player #0 SamXon (IP:PORT) connected
Player #0 SamXon - BE GUID: 534aefe2ee481dc835b2729baca66e07
Connected to BE Master
WORLD        : UpdateEntities
 RPL          : rpl::Pip::ProcessNetToGame
  SCRIPT    (W): Ban Settings could not be loaded from BE Storage: Relying on default config
WORLD        : UpdateEntities
 WORLD        : Frame`



`WORLD        : UpdateEntities
 WORLD        : Frame
  ENTITY       : SpawnEntityPrefab @"{23ADBBC31B6A3DC6}Prefabs/Characters/Factions/OPFOR/USSR_Army/Character_USSR_AR.et"
   WORLD        : InitEntities 'world'
    ENTITY       : Init entity @"ENTITY:4611686018427390137" ('SCR_ChimeraCharacter','Assets/Characters/Basebody/Basebody_Male_01.xob') at <5778.045898 217.250504 7060.766113> @"{23ADBBC31B6A3DC6}Prefabs/Characters/Factions/OPFOR/USSR_Army/Character_USSR_AR.et"
     INVENTORY (E): Couldn't insert item @"ENTITY:4611686018427390173" ('GenericEntity','Assets/Weapons/Magazines/RPK74/Magazine_45rnd_RPK74.xob') at <0.000000 0.000000 0.000000> @"{D78C667F59829717}Prefabs/Weapons/Magazines/Magazine_545x39_RPK_45rnd_4Ball_1Tracer.et" into storage @"ENTITY:4611686018427390152" ('GenericEntity','Assets/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.xob') at <5777.917480 218.366180 7060.869141> @"{B6EEF03975F21E4E}Prefabs/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.et" while spawning initial items inside the storage.
     INVENTORY (E): Couldn't insert item @"ENTITY:4611686018427390174" ('GenericEntity','Assets/Weapons/Magazines/RPK74/Magazine_45rnd_RPK74.xob') at <0.000000 0.000000 0.000000> @"{D78C667F59829717}Prefabs/Weapons/Magazines/Magazine_545x39_RPK_45rnd_4Ball_1Tracer.et" into storage @"ENTITY:4611686018427390152" ('GenericEntity','Assets/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.xob') at <5777.917480 218.366180 7060.869141> @"{B6EEF03975F21E4E}Prefabs/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.et" while spawning initial items inside the storage.
     INVENTORY (E): Couldn't insert item @"ENTITY:4611686018427390175" ('GenericEntity','Assets/Weapons/Magazines/RPK74/Magazine_45rnd_RPK74.xob') at <0.000000 0.000000 0.000000> @"{D78C667F59829717}Prefabs/Weapons/Magazines/Magazine_545x39_RPK_45rnd_4Ball_1Tracer.et" into storage @"ENTITY:4611686018427390152" ('GenericEntity','Assets/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.xob') at <5777.917480 218.366180 7060.869141> @"{B6EEF03975F21E4E}Prefabs/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.et" while spawning initial items inside the storage.
     INVENTORY (E): Couldn't insert item @"ENTITY:4611686018427390176" ('GenericEntity','Assets/Weapons/Magazines/RPK74/Magazine_45rnd_RPK74.xob') at <0.000000 0.000000 0.000000> @"{D78C667F59829717}Prefabs/Weapons/Magazines/Magazine_545x39_RPK_45rnd_4Ball_1Tracer.et" into storage @"ENTITY:4611686018427390152" ('GenericEntity','Assets/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.xob') at <5777.917480 218.366180 7060.869141> @"{B6EEF03975F21E4E}Prefabs/Items/Equipment/Accessories/Pouch_Soviet_45rnd_RPK74/Pouch_Soviet_45rnd_RPK74.et" while spawning initial items inside the storage.
WORLD        : UpdateEntities
 WORLD        : Frame
  DEFAULT   (W): DamageManager.HandleDamage - Struck HitZone doesn't belong to this damage manager! Prefab: @"ENTITY:813922" ('GenericEntity','Assets/Structures/Signs/Warnings/Sign_RectangleUp_A.xob') at <5756.173340 219.993515 7077.151855>
RPL       (E): RpcError: Calling a RPC from an unregistered item! itemType='script::Game::SCR_BudgetEditorComponent', rpc='OnEntityCoreBudgetUpdatedOwner'
Player #0 SamXon disconnected`

**Comment by Greelan** [comment]
Can you put that in three backticks at start and end?

**Comment by sxon-xo** [comment]
> Can you put that in three backticks at start and end?

See if that is better. My apologies.

**Comment by Greelan** [comment]
Thanks. That's from the AMP logs rather than just the console?

**Comment by sxon-xo** [comment]
> Thanks. That's from the AMP logs rather than just the console?

Yes sir. Those are the amplogs in the instance. Would you like me to copy and paste straight from the console, or does that give you everything you need?

**Comment by Greelan** [comment]
The amp logs is what is important, thanks
